packages/lab/src/stepper-input/useStepperInput.ts

```diff
@@ -192,19 +167,15 @@ export const useStepperInput = (
     }
   };
 
-  const handleInputFocus = () => {
-    setHasAnnounced(false);
-  };
-
-  const handleInputChange = (event: KeyboardEvent) => {
-    const changedValue = (event.currentTarget as HTMLInputElement).value;
+  const handleInputChange = (event: ChangeEvent<HTMLInputElement>) => {
+    const changedValue = (event.target as HTMLInputElement).value;
```

Comment:
Little weird to see a cast is needed given `event` is typed to input element

Reply:
updated